lifecycle/lifecycle-livedata-core-ktx-lint/src/test/java/androidx/lifecycle/lint/NonNullableMutableLiveDataDetectorTest.kt

```diff
@@ -760,4 +765,40 @@ Fix for src/com/example/MyClass2.kt line 30: Change `LiveData` type to nullable:
             ).indented()
         ).expectClean()
     }
+
+    // https://issuetracker.google.com/issues/184830262
```

Comment:
You can remove the comment and just add that bug as another `Fixes: ` line for the PR.

Reply:
Description is updated and comment is removed.